guide/items.md

```diff
@@ -1,5 +1,16 @@
 ## Items
 
+`extern crate` statements must be first in a file. They must be ordered
```

Comment:
There is already an [Extern crate] section. These two sentences should probably be placed there.

[Extern crate]: https://github.com/rust-lang-nursery/fmt-rfcs/blob/86720261f2bd1477062bf584ddcae6c859f90bfd/guide/items.md#extern-crate